akka-persistence-query/src/main/scala/akka/persistence/query/typed/EventEnvelope.scala

```diff
@@ -58,7 +82,21 @@ final class EventEnvelope[Event](
     val timestamp: Long,
     val eventMetadata: Option[Any],
     val entityType: String,
-    val slice: Int) {
+    val slice: Int,
+    val filtered: Boolean) {
```

Comment:
Doh, this is still not covering it. We still want to use `eventEnvelope=None` for the filtered events, so can still not see that it is from backtracking or not

Reply:
Could we do an additional set/list of name of "flags" for producers and consumers to use if they want?

Reply:
I added a `source: String` field, see usage in https://github.com/akka/akka-persistence-r2dbc/pull/348/commits/b73eba234d2a62701bd9c958e825d40f47106417